etc/profile-a-l/funnyboat.profile

```diff
@@ -7,12 +7,13 @@ include globals.local
 
 noblacklist ${HOME}/.funnyboat
 
-include disable-common.inc
-include disable-devel.inc
 ignore noexec /dev/shm
-include disable-exec.inc
 include allow-python2.inc
 include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
 include disable-interpreters.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
```

Comment:
~`include disable-write-mnt.inc`~ (`disable-mnt`)

Reply:
Not sure I understand this one. Do you suggest to drop `disable-mnt` and add `include disable-write-mnt.inc`?

Reply:
Doh, now I see what you mean. Fixed!